--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -709,7 +709,7 @@ configure_file(${PROJECT_SOURCE_DIR}/bui
   @ONLY
 )
 
-set(ConfigPackageLocation lib/cmake/Qhull)
+set(ConfigPackageLocation ${LIB_INSTALL_DIR}/cmake/Qhull)
 install(EXPORT QhullTargets
   FILE
     QhullTargets.cmake
@@ -728,7 +728,7 @@ install(
     Devel
 )
 
-set(PkgConfigLocation lib/pkgconfig)
+set(PkgConfigLocation ${LIB_INSTALL_DIR}/pkgconfig)
 foreach(pkgconfig IN ITEMS "${qhull_SHAREDR};Qhull reentrant shared library"
                            "${qhull_STATIC};Qhull static library"
                            "${qhull_STATICR};Qhull reentrant static library"
