--- qhull-2012.1/src/testqset/testqset.c.orig	2012-02-13 01:15:19.043981137 -0200
+++ qhull-2012.1/src/testqset/testqset.c	2012-02-13 01:15:29.331024069 -0200
@@ -214,7 +214,7 @@ void readOptions(int argc, char **argv,
     char *endp;
 
     if (argc != 2 && argc != 3) {
-        printf(promptstr);
+        printf("%s", promptstr);
         exit(0);
     }
     numIntsArg= strtol(argv[1], &endp, 10);
